src/autoscaler/metricscollector/config/config.go

```diff
@@ -23,11 +24,13 @@ var defaultCfConfig = cf.CfConfig{
 }
 
 type ServerConfig struct {
-	Port int `yaml:"port"`
+	Port      int  `yaml:"port"`
+	EnableSSL bool `yaml:"enable_ssl"`
```

Comment:
Why isn't the SSL parts just here as an option? If present, we do SSL, if not, we don't.